experiment/run_experiment.py

```diff
@@ -267,11 +275,8 @@ def start_dispatcher(config: Dict, config_dir: str):
     """Start the dispatcher instance and run the dispatcher code on it."""
     dispatcher = get_dispatcher(config)
     # Is dispatcher code being run manually (useful for debugging)?
-    manual_experiment = os.getenv('MANUAL_EXPERIMENT')
-    if not manual_experiment:
-        dispatcher.create_async()
     copy_resources_to_bucket(config_dir, config)
-    if not manual_experiment:
+    if not os.getenv('MANUAL_EXPERIMENT'):
```

Comment:
Can this be a helper in experiment_utils.py ?

Reply:
I don't see a need for this, it's used pretty infrequently.